doc/source/gitwash/development_workflow.rst

```diff
@@ -77,24 +77,24 @@ what the changes in the branch are for.  For example ``add-ability-to-fly``, or
     # Update the mirror of trunk
     git fetch upstream
     # Make new feature branch starting at current trunk
-    git branch my-new-feature upstream/main
-    git checkout my-new-feature
+    git branch cool-new-feature upstream/main
```

Comment:
This line is no longer needed.

Reply:
I'd replace both lines (this one and the one below) with the equivalent command
```
git switch -c cool-new-feature upstream/main
```
from the man page
> `git switch [<options>] (-c|-C) <new-branch> [<start-point>]`


Note: that the `-n` option in the line below doesn't exist.